Day2/BoxFactory.py

```diff
@@ -0,0 +1,30 @@
+class BoxFactory:
+    
+    packaging_dimensions = None
+    finished_packaging = []
+    
+    def __init__(self, packaging_dimensions):
+        self.packaging_dimensions = packaging_dimensions
+        self.finished_packaging = []
+
+    def make_box(self, length, width, height):
+        side1 = length*width
+        side2 = width*height
+        side3 = height*length
+        return (side1, side2, side3)
+    
+    def make_all_boxes(self):
+        for box in self.packaging_dimensions:
+            box_with_sides = self.make_box(*box)
+            self.finished_packaging.append(box_with_sides)
+    
+    def make_cylinder(self, radius, height):
```

Comment:
cylinder IS A LIE cylinder nem BOX mifaszt csinal itt :D
legyen egy packaging class v shapes etc

egy rendes autogyarban nem fogsz tuni kenyeret venni de meg motorkerekpart sem
elmesz a tesla gyarba ott teslat gyartanak nem mast
teslabol gyartanak tobbfelet de mind tesla
box lehet teglatest lehet kocka ami nem lehet az gomb meg henger